tests/test_argx.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2015, Aiven, https://aiven.io/
+#
+# This file is under the Apache License, Version 2.0.
+# See the file `LICENSE` for details.
+
+from aiven.client.argx import CommandLineTool, OutputFormats
+
+import pytest
+
+pytestmark = [pytest.mark.unittest, pytest.mark.all]
+
+
+@pytest.mark.parametrize(
+    "value", [
+        1,
+        "a string",
+        [1, 2, 3, 4, 5],
+        [(1, 2), (3, 4), (5, 6)],
+        ["a", "bunch", "of", "strings"],
+        ["strings", "and", 1337, "numbers"],
+        {
+            "this": "is",
+            "a": "dict"
+        },
+        [
+            {
+                "nested": {
+                    "a": 1,
+                    "b": 2,
+                }
+            },
+            {
+                "nested": {
+                    "a": 3,
+                    "b": 5,
+                }
+            },
+        ],
+        [{
+            "consistent": "dictionary"
+        }, {
+            "consistent": "keys"
+        }],
+        [{
+            "consistent": "dictionary"
+        }, {
+            "inconsistent": "keys"
+        }],
+    ]
+)
+@pytest.mark.parametrize("fmt", list(OutputFormats))
+def test_print_response(request, value, fmt):
+    cli = CommandLineTool(request.node.name)
+    cli._output_format = fmt  # pylint: disable=protected-access
+    print("\n")
+    cli.print_response(value)
```

Comment:
This test doesn't check the resulting output. I think checking the result is necessary for such big patches like this one.